tests/bluetooth/tester/src/audio/btp_pbp.c

```diff
@@ -166,13 +169,26 @@ static int pbp_broadcast_source_adv_setup(void)
 	ext_ad[2].data_len = pba_buf.len;
 	ext_ad[2].data = pba_buf.data;
 
-	err = tester_gap_create_adv_instance(&param, BTP_GAP_ADDR_TYPE_IDENTITY, ext_ad,
-					     ARRAY_SIZE(ext_ad), NULL, 0, &gap_settings, &ext_adv);
-	if (err) {
-		LOG_ERR("Could not set up extended advertisement: %d", err);
-		return -EINVAL;
+	struct btp_bap_broadcast_local_source *source;
+
+	source = btp_bap_broadcast_local_source_from_src_id_get(0);
+
+	if (!source->ext_adv) {
```

Comment:
```suggestion
	if (source->ext_adv == NULL) {
```


Rule 85 from https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html does not allow using non-bools for `if` statements